CONTRIBUTING.md

```diff
@@ -0,0 +1,389 @@
+# How to contribute to Arcs
+
+Thank you for taking the time to contribute to the Arcs project. 
+This guide outlines how to become an Arcs developer.
+
+**Table of Contents**
+
+[Workflow](#workflow)
+* [Filing Bugs](#filing-bugs)
+* [Contributing a Patch](#contributing-a-patch)
+
+[Developer Installation](#developer-installation)
+* [Additional Bazel Installation](#additional-bazel-installation)
+* [Windows Installation Notes](#windows-installation-notes)
+* [Mac Installation Notes](#mac-installation-notes)
+* [IDE Setup](#ide-setup)
+* [Git Setup](#git-setup)
+
+[Starting Arcs](#starting-arcs)
+
+[Testing](#testing)
+* [WebAssembly tests](#webassembly-tests)
+* [Debugging tests](#debugging-tests)
+  * [Debugging unit tests in Chrome](#debugging-unit-tests-in-chrome)
+* [Debugging WebDriver Failures](#debugging-webdriver-failures)
+  * [Graphical (non-headless)](#graphical-non-headless)
+  * [Attaching a Debugger](#attaching-a-debugger)
+
+
+## Workflow
+
+### Filing Bugs
+
+**If you notice a security issue, please email 
+[arcs-team@google.com](mailto:arcs-team+security@google.com) instead of filing a bug.**
+
+TODO(arcs-team): Devise a way to track bugs internally and in the open.
+
+### Contributing a Patch
+
+1) Create a PR and go through reviews.
+
+   If you are an external contributor, you will have to agree to the 
+   [Contributor License Agreement](https://opensource.google/docs/cla/).
+
+1) When you are ready to merge, add the "ready to pull" label to your PR.
+
+   This will trigger an internal review process for the change. If you are an 
+   internal contributor, please click `+2` to the Safe Review. Otherwise, please
+   reach out to a Googler to help you land the change.
```

Comment:
We should replace "Googler" with "approved committers" here and rest of PR, and point to a file with a list of who they are by GitHub account. That file may not exist today, if so feel free to put a TODO to create it and link to it.